src/test/resources/smarthHouseFeatures/Login.feature

```diff
@@ -0,0 +1,10 @@
+# Login feature review the behavior in thie feature
+Feature: SHLogin
+
+  Scenario: Home page is displayed once set credential in login page
+    Given 'SMARTHOUSE s page is loaded
+    And click on button 'Login' on the header
+    And set username for 'admin'
+    And set password for 'admin'
+    And click on button 'Login'
+    And click on button 'x' on header modal
```

Comment:
According to the test case it should be:
Given I loging to 'SMARTHOUSE' page with properly credentials